.deploy/nais-preprod.yaml

```diff
@@ -41,7 +41,11 @@ spec:
       enabled: true
       claims:
         groups:
-          - id: "928636f4-fd0d-4149-978e-a6fb68bb19de"  # 0000-GA-STDAPPS gir tilgang til prosessering
```

Comment:
Tja - vi kan ha med denne for å gjøre dette litt prodlikt? Denne er en enkel erstatning for medlemskap i teamfamilie-gruppen.